interface.go

```diff
@@ -250,6 +250,17 @@ type LoopInSwapInfo struct { // nolint
 	HtlcAddressNP2WSH btcutil.Address
 }
 
+// LoopOutSwapInfo contains essential information of a loop-out swap after the
+// swap is initiated.
+type LoopOutSwapInfo struct { // nolint
```

Comment:
nit: `// nolint:golint` rather than give this struct a full linter free pass.

Reply:
fixed